penguin/tools/editing/legacy_multifile.py

```
@@ -91,28 +84,18 @@ def _build_unified_patch(self, edits: List[FileEdit]) -> str:
-            ):
+            if not edit.applied or edit.original_bytes is None:
                 continue
             try:
-                shutil.copy2(edit.backup_path, edit.file_path)
+                Path(edit.file_path).write_bytes(edit.original_bytes)
                 edit.applied = False
             except Exception:
                 success = False
         return success
 
     def cleanup_backups(self, edits: List[FileEdit], keep_backups: bool = True) -> None:
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)